lib/Doctrine/DBAL/Driver/Mysqli/MysqliConnection.php

```diff
@@ -135,6 +136,12 @@ public function getServerVersion()
      */
     public function requiresQueryForServerVersion()
     {
+        Deprecation::triggerIfCalledFromOutside(
```

Comment:
This method seems to be called only from tests, should be `trigger()` (the same is for the rest of the implementations).

Reply:
`Connection::getServerVersion` calls it